cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -43,3 +70,37 @@ def test_non_admin_user_reports_access_rest(appliance, request, rbac_api):
     report_data = rbac_api.collections.reports.all
     assert_response(appliance)
     assert len(report_data)
+
+
+@pytest.mark.tier(1)
+def test_reports_custom_tags(appliance, request, create_custom_tag):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: low
+        initialEstimate: 1/3h
+        setup:
+            1. Add custom tags to appliance using black console
+                i. ssh to appliance, vmdb; rails c
+                ii. cat = Classification.create_category!(
+                    name: "rocat1", description: "read_only cat 1", read_only: true)
+                iii. cat.add_entry(name: "roent1", description: "read_only entry 1")
+        testSteps:
+            1. Create a new report with the newly created custom tag/category.
+        expectedResults:
+            1. Report must be created successfully.
+    """
+    category_name = create_custom_tag
+    report_data = {
+        "menu_name": "Custom Category Report {}".format(category_name),
+        "title": "Custom Category Report Title {}".format(category_name),
+        "base_report_on": "Availability Zones",
+        "report_fields": [
+            "Cloud Manager.My Company Tags : description_{}".format(category_name),
+            "VMs.My Company Tags : description_{}".format(category_name),
+        ],
+    }
+    report = appliance.collections.reports.create(**report_data)
+    request.addfinalizer(report.delete)
```

Comment:
Question: Able to use `delete_if_exists` here?

Reply:
Same comment as I mentioned [here](https://github.com/ManageIQ/integration_tests/pull/8697#discussion_r275235853).